openlibrary/templates/merge_queue/merge_queue.html

```diff
@@ -34,158 +40,166 @@ <h1>$_('Community Edit Requests')</h1>
+              <div class="mr-details__comment-preview">
+                $ latest_comment = comments[-1]['message'] if comments else _('No comments yet.')
+                $latest_comment
               </div>
+              $# End : Most recent comment preview
 
-              $if len(comments) > 1:
-                <div class="comment-cell__description">
-                  <span>$_('Showing most recent comment only.')</span>
-                  <a href="javascript:;" class="comment-expand" data-target-id="hidden-comments-$r['id']">$_('View all')</a>
+              $# Start : Comment section
+              $# XXX : Replace this with a modal form
+              <div class="comment-panel hidden">
+                <div class="comment-panel__comment-display">
+                  $for c in comments:
+                    <div class="comment-panel__comment"><span class="commenter">@$c['username']</span> $c['message']</div>
                 </div>
-              <div id="hidden-comments-$r['id']" class="comment-cell__old-comments hidden">
-                $for c in comments[:-1]:
-                  $:render_template('merge_queue/comment', comment=c)
-              </div>
-              <div class="comment-cell__newest-comment">
-                $if comments:
-                  $:render_template('merge_queue/comment', comment=comments[-1])
-                $else:
-                  <span>$_('No comments yet.')</span>
+                $if is_submitter or can_merge:
+                  <div class="comment-panel__input">
+                    $# XXX: Ensure that this has a drag handle
+                    <textarea class="comment-panel__reply-input" rows="1" placeholder="$_('Add a comment...')"></textarea>
+                    <input class="comment-panel__reply-btn" type="button" value="$_('Reply')">
+                  </div>
               </div>
-              $if is_submitter or can_merge:
-                <hr>
-                <div class="comment-cell__input">
-                  <textarea rows="1" placeholder="$_('Add a comment...')"></textarea>
-                  <input class="mr-comment-btn" type="button" value="Reply" data-mrid="$r['id']" data-merge-type="$r['mr_type']">
-                </div>
-            </td>
-            <td id="reviewer-cell-$(r['id'])">
-              $if can_merge and is_open and r.get('reviewer'):
-                $r['reviewer'] <span class="mr-unassign" data-mrid="$r['id']" data-merge-type="$r['mr_type']">&times;</span>
-                <br>
-                $datestr(r['updated'])
-              $elif r.get('reviewer'):
-                $r['reviewer']
-            </td>
-            <td>
+              $# End : Comment section
+
+              $# Start : Submitted by line
+              <span class="mr-details__opened-by">
+                $:_('MR #%(id)s opened %(date)s by <span class="commenter">@%(submitter)s</span>', id=r['id'], date=datestr(r["created"]), submitter=r["submitter"])
```

Comment:
The username here looks like a link but isn't clickable ; can we make it a link ? 
![image](https://github.com/internetarchive/openlibrary/assets/6251786/be4b7122-f784-448a-a979-7ca12dd59471)